gpflow/config/__config__.py

```diff
@@ -37,6 +37,7 @@
 * ``GPFLOW_FLOAT``: "float16", "float32", or "float64"
 * ``GPFLOW_POSITIVE_BIJECTOR``: "exp" or "softplus"
 * ``GPFLOW_POSITIVE_MINIMUM``: Any positive float number
+* ``LIKELIHOODS_POSITIVE_MINIMUM``: Any positive float number
```

Comment:
We should probably prepend `GPFLOW_` for consistency with the above...

```suggestion
* ``GPFLOW_LIKELIHOODS_POSITIVE_MINIMUM``: Any positive float number
```